labs/memory_bound/loop_interchange_1/src/lib.rs

```diff
@@ -36,8 +41,8 @@ pub fn multiply(result: &mut Matrix, a: &Matrix, b: &Matrix) {
     zero(result);
 
     for i in 0..N {
-        for j in 0..N {
-            for k in 0..N {
+        for k in 0..N {
+            for j in 0..N {
```

Comment:
Could you explain this change? Did I copy it wrong from the C++?

Reply:
thats just me accidentally pushing the solved version

Reply:
fixed!